### PR TITLE
PIM-5109: add trash icons

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

Little fixes on the peb

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | NA
| Added Behats                      | NA
| Changelog updated                 | N
| Review and 2 GTM                  | N
| Micro Demo to the PO (Story only) | N
| Migration script                  | NA
| Tech Doc                          | NA

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/filter/attribute/metric.html [hunk 1]
```diff
@@ -10,9 +10,10 @@
     <input type="text" name="filter-data" class="value" value="<%- value.data %>" <%- editable ? '' : 'disabled' %>>
 
     <select class="select2 unit" name="filter-unit" <%- editable ? '' : 'disabled' %>>
-        <% _.each(units.units, function (unitChoice, key) { %>
+        <% console.log(units) %>
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/filter/attribute/metric.html]: remove this

**Comment by phaseinducer** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/export/product/edit/content/structure/attributes-selector.js [hunk 2]
```diff
@@ -33,7 +33,8 @@ define(
             events: {
                 'click .attribute-groups li': 'changeAttributeGroup',
                 'keyup .search-field': 'updateSearch',
-                'click .clear': 'clear'
+                'click .clear': 'clear',
+                'click li .remove': 'removeAttribute'
```
Comment (SamirBoulil) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/export/product/edit/content/structure/attributes-selector.js]: Sure you need li

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/export/product/edit/content/structure/attributes-selector.js [hunk 3]
```diff
@@ -236,6 +237,22 @@ define(
...
+            /**
+             * Remove the clicked row from the selected attributes
+             *
+             * @param {event} event
+             */
+            removeAttribute: function (event) {
+                var element = $(event.currentTarget).parents('li');
+                var selected = this.getSelected();
+
+                selected = _.without(selected, element.data('attributeCode'));
```
Comment (SamirBoulil) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/export/product/edit/content/structure/attributes-selector.js]: If you reset the selected attributes, this should be plural

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/export/product/edit/content/structure/attributes-selector.js [hunk 4]
```diff
@@ -236,6 +237,22 @@ define(
...
+
+                this.setSelected(selected);
+
+                this.$('.attributes > div ul').append(element);
```
Comment (SamirBoulil) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/export/product/edit/content/structure/attributes-selector.js]: How about rerendering the attributes with the state ?

**Comment by SamirBoulil** [comment]
GTM

**Pull request merged**